app/src/main/java/cm/aptoide/pt/presenter/MainPresenter.java

```diff
@@ -7,7 +7,7 @@
 
 import android.content.SharedPreferences;
 import cm.aptoide.pt.AptoideApplication;
-import cm.aptoide.pt.autoupdate.AutoUpdateManager;
+import cm.aptoide.pt.autoupdate.kotlin.AutoUpdateManager;
```

Comment:
we should not organise classes packages by language, this could be under autoupdate package.